valio-master/Valio/ScheduleTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ScheduleTableViewController.swift
+//  Valio
+//
+//  Created by Sam Soffes on 6/5/14.
+//  Copyright (c) 2014 Sam Soffes. All rights reserved.
+//
+
+import UIKit
+
+class ScheduleTableViewController: UITableViewController {
+
+	lazy var 📅: NSArray = {
+		let path = NSBundle.mainBundle().pathForResource("valio", ofType: "json")!
+		let data = NSData(contentsOfFile: path, options: nil, error: nil)!
```

Comment:
This reads data from disk on main thread, I would dispatch that to a background queue.